src/ai/backend/web/server.py

```diff
@@ -552,6 +552,13 @@ async def server_main(
     app.on_shutdown.append(server_shutdown)
     app.on_cleanup.append(server_cleanup)
 
+    async def on_prepare(request, response):
+        # Remove "Server" header for a security reason.
+        if "Server" in response.headers:
+            del response.headers["Server"]
```

Comment:
Just a minor refactor:
```suggestion
        response.headers.popall("Server", None)
```